Framework/DataHandling/src/LoadNexusProcessed.cpp

```diff
@@ -1497,7 +1742,15 @@ API::Workspace_sptr LoadNexusProcessed::loadEntry(NXRoot &root,
   }
 
   if (mtd_entry.containsGroup("peaks_workspace")) {
-    return loadPeaksEntry(mtd_entry);
+    try {
+      // try standard PeakWorkspace first
+      return loadPeaksEntry(mtd_entry);
+    } catch (std::exception &err) {
+      g_log.notice("Standard PeakWorkspace Load failed");
+      g_log.information(err.what());
+      g_log.notice("¯\\_(ツ)_/¯ LeanElasticPeakWorkspace?");
```

Comment:
I don't think you need to log any messages here, you can just silently move to the other loader.

Reply:
okay, I will remove these log messages